cmake/public/cuda.cmake

```diff
@@ -291,7 +291,7 @@ if(CAFFE2_STATIC_LINK_CUDA AND NOT WIN32)
     set_property(
         TARGET caffe2::cublas PROPERTY INTERFACE_LINK_LIBRARIES
         "${CUDA_TOOLKIT_ROOT_DIR}/lib64/libcublas_static.a")
-    if (CUDA_VERSION VERSION_EQUAL 10.1)
+    if (CUDA_VERSION VERSION_EQUAL 10)
```

Comment:
Pretty sure this is the last missing piece of the puzzle.

@ezyang or @soumith can you give me some context on what this is actually doing and why it's specific to just CUDA 10+

Reply:
I see it was originally added here:

https://github.com/pytorch/pytorch/pull/26823/files#diff-e1f6ba8d0726bd4dc7d55fa517bd381cR268-R272

Reply:
Well, it looks to me like `libcublasLt_static.a` doesn't necessarily exist on all versions of CUDA, so the test here is probably just trying to test for whether or not you're a version of CUDA that has it or not. But Mingbo would know best (and he's unfortunately not here lol)